src/app/shared/entry-wizard/entry-wizard.component.html

```diff
@@ -0,0 +1,38 @@
+<mat-progress-bar *ngIf="isLoading$ | async" mode="indeterminate"></mat-progress-bar>
+<div>
+  <p>Toggle repositories on to create workflows from the associated Git repository. Once added, these workflows can be customized.</p>
+  <p>This approach is best for Git repositories with only one workflow.</p>
```

Comment:
Slightly more broadly, is it just best for the first workflow in a repo? 
Or do we really mean that if a repo has more than one, this wizard should be avoided?

Reply:
The idea is that if you have only one workflow in a repository, you probably don't need to specify things like the workflow name.

If you have multi workflows in one repository, each workflow will have a workflow name. Chances are there will not be a main repository that doesn't use a workflow name.